lib/sprockets/commoner/processor.rb

```diff
@@ -60,12 +60,13 @@ def self.configure(env, *args, **kwargs)
         env.register_postprocessor('application/javascript', self.new(env.root, *args, **kwargs))
       end
 
-      attr_reader :include, :exclude, :babel_exclude
-      def initialize(root, include: [root], exclude: ['vendor/bundle'], babel_exclude: [/node_modules/])
+      attr_reader :include, :exclude, :babel_exclude, :extra_options
+      def initialize(root, include: [root], exclude: ['vendor/bundle'], babel_exclude: [/node_modules/], extra_options: [])
```

Comment:
It takes any kind of iterable that yields (path, hash). Could maybe add a rubydoc